src/dotnetTestExplorer.ts

```diff
@@ -69,12 +69,12 @@ export class DotnetTestExplorer implements TreeDataProvider<TestNode> {
 
     public runAllTests(): void {
         this.evaluateTestDirectory();
-        Executor.runInTerminal("dotnet test", this.testDirectoryPath);
+        Executor.runInTerminal(`dotnet test -v=q ${this.enableBuild()} ${this.enableRestore()}`, this.testDirectoryPath);
         AppInsightsClient.sendEvent("runAllTests");
     }
 
     public runTest(test: TestNode): void {
-        Executor.runInTerminal(`dotnet test --filter FullyQualifiedName~${test.fullName}`, this.testDirectoryPath);
+        Executor.runInTerminal(`dotnet test -v=q ${this.enableBuild()} ${this.enableRestore()} --filter FullyQualifiedName~${test.fullName}`, this.testDirectoryPath);

```

Comment:
One minor issue is that there will be one more extra space if there is empty `${this.enableBuild()} ${this.enableRestore()}`:
`dotnet test -v=q   --filter FullyQualifiedName~xunit2.UnitTest1.Test1`